src/courseware/course/celebration/CelebrationModal.scss

```diff
@@ -22,7 +22,6 @@
     justify-content: center;
 
     button {
-      @extend .btn-primary;
       font-size: 1.2rem;
```

Comment:
[question]: Do we want to create an alternative to `.btn-primary` so that the button has a primary color?

Reply:
This button is already primary
```
<Button onClick={onClose}>{intl.formatMessage(messages.forward)}</Button>
```